README.md

```diff
@@ -56,3 +56,18 @@ You can also use multi-line syntax using `"""` for larger templates:
       }
     """
 ```
+
+### Escaping Characters
+
+Including a literal a closing brace inside the text provided by a snippet's tab stop will close
+that tab stop early. You need to escape it -- with *two* backslashes, like this:
```

Comment:
Does `To prevent that, escape the brace with two backslashes, like so:` sound better to you?